src/main/java/makePost/drivers/MakePostDatabaseAccess.java

```diff
@@ -0,0 +1,193 @@
+package makePost.drivers;
+
+import com.opencsv.exceptions.CsvException;
+import makePost.ui.MakePostView;
+import makePost.use_case.MakePostDataAccessInterface;
+
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.CSVWriter;
+
+import java.io.*;
+import java.util.*;
+
+public class MakePostDatabaseAccess implements MakePostDataAccessInterface {
+
+    @Override
+    public int getNumberOfPosts(){
+        String filePath = "src/main/java/Database/numPostsCreated.csv";
```

Comment:
I think we should pass the filepaths for different files to the data access as arguments in the constructor. This way, if something needs to change when we are exporting our work, we'll need to change next code

Reply:
Done!